packages/aws-cdk-lib/aws-ec2/lib/instance.ts

```diff
@@ -315,6 +315,14 @@ export interface InstanceProps {
    * @default false
    */
   readonly ebsOptimized?: boolean;
+
+  /**
+   * Indicates whether an instance stops or terminates when you initiate shutdown from the instance
+   * (using the operating system command for system shutdown).
+   *
```

Comment:
It would be good to add the doc link, like the [LaunchTemplateProps](https://github.com/aws/aws-cdk/blob/v2.142.1/packages/aws-cdk-lib/aws-ec2/lib/launch-template.ts#L318).

```suggestion
   * Indicates whether an instance stops or terminates when you initiate shutdown from the instance
   * (using the operating system command for system shutdown).
   *
   * @see https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/terminating-instances.html#Using_ChangingInstanceInitiatedShutdownBehavior
   *
```